README.md

```diff
@@ -1 +1 @@
-# fetch-with-redirected-login
+# Vouch redirect Plugin for Backstage
```

Comment:
Is this definitely specific for vouch?

Can we fill this out a little?

e.g.

Explain what it does.
Explain that the following settings are required in vouch and how to set them:
```
  vouch:
    cookie:
      sameSite: none
    domains: ["hostofsite", ...domains]
```
Explain what the javascript interface is and how to include it in a project. e.g.
```
import { fetchVouchProtectedResource } from 'fetch-with-redirected-login'
fetchVouchProtectedResource('https://protected-service/hello/world')
```

Reply:
I had to add vouch-specific things there to enable auto-closing feature (redirecting popup through our own site first)

Reply:
Added a readme file